### ISSUE TITLE
Plugin thinks alarm is armed when it isn't

### ISSUE THREAD

**Issue by tagyoureit** [bryanbartow/homebridge-alarm.com#48]
Thanks for updating the ability to turn on/off extra features!  I reinstalled the latest but for some reason the alarm is showing "armed - home" when the alarm is off.  I turned on debugging and also saw
```
  Accessory [Homebridge] Got Characteristic "Security System Current State" value: 0 +2s
  Accessory [Homebridge] Got Characteristic "Security System Target State" value: 0 +3ms
```

I'm assuming 0 = off?  Any thoughts?

**Comment by bryanbartow** [comment]
@tagyoureit I have not experienced this. Anyone else?

**Comment by tagyoureit** [comment]
Ok, so this was a case of the wrong username being entered into the config.json.  So maybe this should turn into an enhancement request where an error is logged if the user can't login.  :man_shrugging:

**Comment by bryanbartow** [comment]
@tagyoureit Then it would be a duplicate of #22 ;)

**Comment by bryanbartow** [comment]
@tagyoureit I just published version 0.7.2, which now logs login errors on alarm.com. You'll also need to update the login call on WrapAPI.

**Comment by tagyoureit** [comment]
👍 Thanks very much!  It's always good to have software idiot-proof.
